reddit_decider/__init__.py

```diff
@@ -990,56 +1006,76 @@ def _minimal_decider(
         )
 
     def make_object_for_context(self, name: str, span: Span) -> Decider:
+        # initialize rust decider from watched manifest file
         rs_decider = None
         try:
             rs_decider = self._filewatcher.get_data()
         except WatchedFileNotAvailableError as exc:
+            experiments_client_counter.labels(
```

Comment:
Probably make sense to extract this into nested function under `make_object_for_context`, e.g.:
```python
def make_object_for_context(self, name: str, span: Span) -> Decider:
  def inc_failure_counter(failure_type: str):
    experiments_client_counter.labels(
      operation="make_object_for_context",
      success="false",
      error_type=failure_type,
      pkg_version=_pkg_version,
    ).inc()
```

Reply:
good call.